Cursive doesn't compile on Windows with termion backend

# Problem description

Compile error:
```
   Compiling cursive v0.9.1
error[E0433]: failed to resolve. Maybe a missing `extern crate signal_hook;`?
  --> C:\Users\agavrilov\.cargo\registry\src\github.com-1ecc6299db9ec823\cursive-0.9.1\src\backend\termion.rs:19:5
   |
19 | use signal_hook::iterator::Signals;
   |     ^^^^^^^^^^^ Maybe a missing `extern crate signal_hook;`?

```

# Environment

* Operating system: windows
* Backend used: termion
* Cursive version: 0.9.1 from crates.io

--- a/src/backend/termion.rs
+++ b/src/backend/termion.rs
@@ -16,6 +16,8 @@ use self::termion::screen::AlternateScreen;
 use self::termion::style as tstyle;
 use crossbeam_channel::{self, Receiver, Sender};
 use libc;
+
+#[cfg(unix)]
 use signal_hook::iterator::Signals;
 
 use backend;

